index.d.ts

```diff
@@ -14,4 +14,8 @@ export interface CircleFlagProps extends DetailedHTMLProps<ImgHTMLAttributes<HTM
  */
 declare function CircleFlag(props: CircleFlagProps): JSX.Element;
 
-export { CircleFlag }
\ No newline at end of file
+export { CircleFlag }
+
+declare var countries: Record<string, boolean>;
+
+export { countries };

```

Comment:
Could you keep only 1 `export` instead of having 2??

Reply:
Yes totally, I've just amended that. We could also add `CountryFlagProps` to the same export, if you prefer them all consolidated together?

Reply:
Sure!, thanks for the idea, could you make that change too?

Reply:
yep, all done!